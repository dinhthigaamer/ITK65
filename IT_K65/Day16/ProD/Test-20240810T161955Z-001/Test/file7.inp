485
810010298 350752545 329275188 -146003636 93172922 983830636 -163318921 -148878181 -159191916 452325727 987991266 564323600 -456609740 -811867026 -690427126 717299284 872013177 462475234 593339258 815325466 420025749 -49120215 374451908 -939492722 -289197104 598863685 441686357 669664685 -673420690 -841419997 -553911080 -773972112 -192972691 162270084 979693840 -148074299 -46583765 -157605605 -266220144 453539559 266465935 -980354921 -96311966 70133955 -565460455 408269867 -967381888 -808156619 -603465194 30390605 -716896891 33742917 461073042 720834238 356355193 -912007300 -16713852 663449774 520436712 -440760017 -334407317 651054670 56927294 -74452735 51668137 -786299140 410170223 707460847 -500509177 228439232 -756978233 99202920 -281371902 -578701881 126791191 -181551742 -763951647 855928891 -250855068 -167692248 714713228 672105404 -616007863 119219523 97253163 829065159 768769354 -794107711 331597419 435323424 -654866496 76437439 -838268456 -36553497 385442071 -354866993 847003710 930350551 257713153 669820930 419659913 -647669103 815358358 -679042007 -550752032 -167091567 -266855359 979451872 -604712924 -717643030 -816181716 817175283 172810901 -245516364 -917031351 -319636200 -60329191 55981090 555073224 928065422 17993526 415411709 -658611703 74421907 862151716 -782390030 963147664 -492377073 -154570101 -52145116 187803553 858844161 66540410 418777176 434025748 -78301978 -319393213 -12393024 542953412 786804661 845642067 754996432 -985617475 -487171947 56142664 203173079 -268261708 -817080591 -511720682 -916796553 -945485050 -113304572 -959675810 -76517173 -927796949 536356109 985232195 -511927520 93205058 -891627679 387985353 249018423 -707229467 396180847 902081055 -54424764 399372518 133534573 624400079 454430730 667679592 703957814 -465060693 -846627979 -161264784 90801286 -450827986 287560054 439961271 -63112533 244676712 473626710 -527410610 921650511 -406421807 -912118617 512500401 -736607769 303074389 -188039493 -81934870 912019762 571102528 -890844692 -195145788 -735660574 642752765 -515800273 -632165236 42878822 -800514166 -329335228 486419828 -359791275 908513174 -755993223 359017865 476849094 -183075179 614802212 950251583 720337144 -303375670 299510026 -2103535 901101844 811783631 851998745 981781286 -296433695 610042288 27129580 -103108347 -942517864 842060508 -588904868 614786155 37455106 832366821 -131327173 -133814997 -829130387 229138050 656245092 907475002 -190994667 845993427 -988964488 724877440 -543919899 490097500 -801440880 -874068847 -630492249 969051599 -34797180 -562207906 -176070316 294476776 183013082 878896059 114777915 552608975 -733432094 691633361 -310202252 132209173 670193662 -875964661 805258595 322918534 -114023086 -308052339 -678434650 -403650489 762324844 -924300815 -694255287 -124654565 -554205584 -937300177 -261176613 -73508458 109432773 211706170 439901215 552439674 94775548 -380977630 -359810165 -253450591 -245950479 577420251 -491368943 587328904 -882567592 354585898 -348274687 -498952996 -164633349 -944896629 560252466 -872222909 -933765770 -689152223 677894554 990637176 -677391813 572550782 834529938 35245081 663687669 585422565 164412867 850052113 796429008 -307345676 -127653313 426960600 211854713 -733687358 251466836 -33491050 -882811238 455544614 774581544 768447941 -241033484 441104324 -493975870 308277556 275268265 935941507 565623004 184451316 440728475 -343899570 247569648 -113271698 -176826579 -854023932 658752217 291522742 614107063 -79084988 362094070 686714668 -82581175 736472406 502016312 575563638 -899308712 503504178 -644271272 -437408397 385778869 -746757839 306275973 -663503743 -431116516 -418708278 -515261493 -988182324 900230931 218258353 -714420102 -241118202 294802602 -937785552 -857537498 660745215 364296623 -933755518 -954074337 592452570 364425878 769489186 -306281072 104794794 226146089 -782262921 -108686170 -908334940 207174520 505502806 -871675350 -418867503 -41453834 -669347636 -929734166 648485686 658304400 -928896030 572996466 905703555 979766062 -857340609 417614771 -610222088 718218769 -653796065 998579465 478181627 381377060 508835731 719044650 -303036036 -23257516 -124205919 -198743485 -292679341 -109429490 453417693 438465957 439503655 -823483003 -581181252 -101838102 634446327 636057474 616884863 -496771256 262323226 -523560586 440482558 -627461034 571102495 691018679 -857719060 860122732 -825440699 -972339305 180800298 214750875 -957741538 257287089 461377357 147488151 523196067 689821295 838706522 -858875495 329181411 -480638226 794537056 996664108 187496535 -680808843 -754586420 -859879832 978095207 437394538 -348747383 953466088 -127885357 183656383 328235168 375028815 842713117 -63883506 335146131 -46127006 984271888 252170912 890337523 541283801 734631005 20762985 813266638 -656480184 193860273 167069475 83073828 -549520501 653497104 -714525570 180112987 224043492 -922769289 20471044 -914000175 917779866 676051182 496061056 -649495633 179250328 -907495561 636884587 -912549640 504673850 608324016 -960399645 93072408 -696361039 252158293 
810010298 350752545 329275188 -146003636 93172922 983830636 -163318921 -148878181 -159191916 452325727 987991266 564323600 -456609740 -811867026 -690427126 717299284 872013177 462475234 593339258 815325466 420025749 -49120215 374451908 -939492722 -289197104 598863685 441686357 669664685 -673420690 -841419997 -553911080 -773972112 -192972691 162270084 979693840 -148074299 -46583765 -157605605 -266220144 453539559 266465935 -980354921 -96311966 70133955 -565460455 408269867 -967381888 -808156619 -603465194 30390605 -716896891 33742917 461073042 720834238 356355193 -912007300 -16713852 663449774 520436712 -440760017 -334407317 651054670 56927294 -74452735 51668137 -786299140 410170223 707460847 -500509177 228439232 -756978233 99202920 -281371902 -578701881 126791191 -181551742 -763951647 855928891 -250855068 -167692248 714713228 672105404 -616007863 119219523 97253163 -317072672 768769354 -794107711 331597419 435323424 -654866496 76437439 -838268456 -36553497 385442071 -354866993 847003710 930350551 257713153 669820930 419659913 -647669103 815358358 -679042007 -550752032 -167091567 -266855359 979451872 -604712924 -717643030 -816181716 817175283 172810901 -245516364 -917031351 -319636200 -60329191 55981090 555073224 928065422 17993526 415411709 -658611703 74421907 862151716 -782390030 963147664 -492377073 -154570101 -52145116 187803553 858844161 66540410 418777176 434025748 -78301978 -319393213 -12393024 542953412 786804661 845642067 754996432 -985617475 -487171947 56142664 203173079 -268261708 -817080591 -511720682 -916796553 -945485050 -113304572 -959675810 -76517173 -927796949 536356109 985232195 -511927520 93205058 -891627679 387985353 249018423 -707229467 396180847 902081055 -54424764 399372518 133534573 624400079 454430730 667679592 703957814 -465060693 -846627979 -161264784 90801286 -450827986 287560054 439961271 -63112533 244676712 473626710 -527410610 921650511 -406421807 -912118617 512500401 -736607769 303074389 -188039493 -81934870 912019762 571102528 -890844692 -195145788 -735660574 642752765 -515800273 -632165236 42878822 -800514166 -329335228 486419828 -359791275 908513174 -755993223 359017865 476849094 -183075179 614802212 950251583 720337144 -303375670 299510026 -2103535 901101844 811783631 851998745 981781286 -296433695 610042288 27129580 -103108347 -942517864 842060508 -588904868 614786155 37455106 832366821 -131327173 -133814997 -829130387 229138050 656245092 907475002 -190994667 845993427 -988964488 724877440 -543919899 490097500 -801440880 -874068847 -630492249 969051599 -34797180 -562207906 -176070316 294476776 183013082 878896059 114777915 552608975 -733432094 691633361 -310202252 132209173 670193662 -875964661 805258595 322918534 -114023086 -308052339 -678434650 -403650489 762324844 -924300815 -694255287 -124654565 -554205584 -937300177 -261176613 -73508458 109432773 211706170 439901215 552439674 94775548 -380977630 -359810165 -253450591 -245950479 577420251 -491368943 587328904 -882567592 354585898 -348274687 -498952996 -164633349 -944896629 560252466 -872222909 -933765770 -689152223 677894554 990637176 -677391813 572550782 834529938 35245081 663687669 585422565 164412867 850052113 796429008 -307345676 -127653313 426960600 211854713 -733687358 251466836 -33491050 -882811238 455544614 774581544 768447941 -241033484 441104324 -493975870 308277556 275268265 935941507 565623004 184451316 440728475 -343899570 247569648 -113271698 -176826579 -854023932 658752217 291522742 614107063 -79084988 362094070 686714668 -82581175 736472406 502016312 575563638 -899308712 503504178 -644271272 -437408397 385778869 -746757839 306275973 -663503743 -431116516 -418708278 -515261493 -988182324 900230931 218258353 -714420102 -241118202 294802602 -937785552 -857537498 660745215 364296623 -933755518 -954074337 592452570 364425878 769489186 -306281072 104794794 226146089 -782262921 -108686170 -908334940 207174520 505502806 -871675350 -418867503 -41453834 -669347636 -929734166 648485686 658304400 -928896030 572996466 905703555 979766062 -857340609 417614771 -610222088 718218769 -653796065 998579465 478181627 381377060 508835731 719044650 -303036036 -23257516 -124205919 -198743485 -292679341 -109429490 453417693 438465957 439503655 -823483003 -581181252 -101838102 634446327 636057474 616884863 -496771256 262323226 -523560586 440482558 -627461034 571102495 691018679 -857719060 860122732 -825440699 -972339305 180800298 214750875 -957741538 257287089 461377357 147488151 523196067 689821295 838706522 -858875495 329181411 -480638226 794537056 996664108 187496535 -680808843 -754586420 -859879832 978095207 437394538 -348747383 953466088 -127885357 183656383 328235168 375028815 842713117 -63883506 335146131 -46127006 984271888 252170912 890337523 541283801 734631005 20762985 813266638 -656480184 193860273 167069475 83073828 -549520501 653497104 -714525570 180112987 224043492 -922769289 20471044 -914000175 917779866 676051182 496061056 -649495633 179250328 -907495561 636884587 -912549640 504673850 608324016 -960399645 93072408 -696361039 252158293 