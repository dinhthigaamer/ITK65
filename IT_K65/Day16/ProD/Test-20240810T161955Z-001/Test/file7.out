7930791550781
